src/ui/screens/NotificationSettingsScreen.tsx

```diff
@@ -57,6 +57,7 @@ const styles = StyleSheet.create({
   },
   infoContainer: {
     flexDirection: 'row',
-    paddingRight: 10,
+    paddingHorizontal: 20,
```

Comment:
This padding is not needed, the parent container already have padding. You can also increase the `scale` in the `Padder` between the `Icon` and the `Text` to 1 to have a bit more space.

Add here `alignItems: 'center'` so the icon is aligned with the text.